srtcore/api.cpp

```diff
@@ -2752,28 +2752,6 @@ CUDTException& CUDT::getlasterror()
    return *s_UDTUnited.getError();
 }
 
-int CUDT::perfmon(SRTSOCKET u, CPerfMon* perf, bool clear)
```

Comment:
Shouldn't removal of perfmon be a subject of a different PR?

Reply:
Ideally yes. But here it is in a different commit. And this commit structure will go directly to master. So the commit for this very change will be preserved.

Reply:
Ah, if so, then no problem.